dockerHashing/src/main/scala/cromwell/docker/registryv2/flows/dockerhub/DockerHubFlow.scala

```diff
@@ -16,6 +16,7 @@ object DockerHubFlow {
   private val DockerHubDefaultRegistry = "index.docker.io"
   private val RegistryHostName = "registry-1.docker.io"
   private val AuthorizationServerHostName = "auth.docker.io"
+  private val DockerHubImplicitRegistryHostName = "docker.io"
```

Comment:
Instead of listing these individually and chaining together `|| host == X`s, can we just make a list and say `if list.contains(host)`?